packages/xod-project/src/patch.js

```diff
@@ -1137,35 +1137,35 @@ export const computeVariadicPins = def(
       return fail('TOO_MANY_VARIADIC_MARKERS', { trace: [patchPath] });
     }
 
-    const outputs = listOutputPins(patch);
-    const outputsCount = outputs.length;
+    const outputs = R.compose(Pin.normalizePinLabels, listOutputPins)(patch);
```

Comment:
Why do we need to call `normalizePinLabels` here?
It looks like this function needs only output pins count. Am I right?

Reply:
They are necessary below for computing `pinLabelsOfNonEqualPinTypes`

Reply:
Yep. But that creates another problem and is out of scope of this fix, so I reverted it back.